Logging started at 2016-12-14 00:41:47
File name: C:\Users\Jeremy\Desktop\Project Reality\.\data\logs\err\1481704907589 - Jeremy - [Wed Dec 14][12.41AM].txt
System.getProperty("os.name"): "Windows 10"

[2016-12-14 01:35:57] com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Table 'sql5148466.hs_users' doesn't exist
[2016-12-14 01:35:57] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2016-12-14 01:35:57] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[2016-12-14 01:35:57] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[2016-12-14 01:35:57] 	at java.lang.reflect.Constructor.newInstance(Unknown Source)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.Util.getInstance(Util.java:381)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3515)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3447)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1951)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2101)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2554)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1761)
[2016-12-14 01:35:57] 	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1021)
[2016-12-14 01:35:57] 	at com.vencillio.core.network.mysql.impl.Highscores.run(Highscores.java:38)
[2016-12-14 01:35:57] 	at java.lang.Thread.run(Unknown Source)
[2016-12-14 01:36:18] com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Table 'sql5148466.hs_users' doesn't exist
[2016-12-14 01:36:18] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2016-12-14 01:36:18] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[2016-12-14 01:36:18] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[2016-12-14 01:36:18] 	at java.lang.reflect.Constructor.newInstance(Unknown Source)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.Util.getInstance(Util.java:381)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3515)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3447)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1951)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2101)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2554)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1761)
[2016-12-14 01:36:18] 	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1021)
[2016-12-14 01:36:18] 	at com.vencillio.core.network.mysql.impl.Highscores.run(Highscores.java:38)
[2016-12-14 01:36:18] 	at java.lang.Thread.run(Unknown Source)
[2016-12-14 01:37:16] java.lang.NumberFormatException: For input string: "school"
[2016-12-14 01:37:16] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-12-14 01:37:16] 	at java.lang.Integer.parseInt(Unknown Source)
[2016-12-14 01:37:16] 	at java.lang.Integer.parseInt(Unknown Source)
[2016-12-14 01:37:16] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:112)
[2016-12-14 01:37:16] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-12-14 01:37:16] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-12-14 01:37:16] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:216)
[2016-12-14 01:37:16] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-12-14 01:37:16] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-12-14 01:37:16] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
